File: src/Challenge_1.java
import java.util.* ;
public class Challenge_1 {
    public static void main ( String[] RGS ) {
        List < String > vendingMachine = new ArrayList <> ( Arrays.asList ( "Cola", "EMPTY", "Chips", "EMPTY", "Candy", "Water", "EMPTY" ) ) ;
        int index = 0 ;
        int count = 0 ;

        for ( String item : vendingMachine ){
            if ( item.equals("EMPTY") ) {
                vendingMachine.set( index , "RESTOCKED ITEM" ) ;
                count ++ ;
            }
            index ++ ;
        }
        System.out.println ( " There were " + count + " empty slots " ) ;
        System.out.println ( " The updated inventory of vending machine is : " ) ;
        count = 0 ;
        for ( String item : vendingMachine ) {
            System.out.println ( " Slot " + ( count += 1 ) + " : " + item ) ;
        }
    }

}

File: src/Challenge_1.kt
 fun main() {
        val vendingMachine =
            mutableListOf("Cola", "EMPTY", "Chips", "EMPTY", "Candy", "Water", "EMPTY")
        var count = 0
        for (index in vendingMachine.indices) {
            if (vendingMachine[index] == "EMPTY") {
                vendingMachine[index] = "Restocked Item"
                count++
            }
        }
        println("The updated Inventory is: ")
        for (item in vendingMachine) {
                println(item)
        }
        println("There were $count empty slots ")
 }

